src/Settings.cpp

```diff
@@ -356,3 +358,10 @@ void Settings::restoreDefaults ()
     settings.clear();
     m_hCache.clear();
 }
+
+QSettings::Format Settings::getDotenvFormat()
+{
+    static const QSettings::Format DotenvFormat = QSettings::registerFormat("env", &DotenvFormat::readEnvFile, nullptr);
+
+    return DotenvFormat;
+}
```

Comment:
I would delete this function or at least move it to some other file. The reason is that the Settings.cpp and Settings.h file were added to have a class which is at small and simple as possible because the settings are queried all the time throughout the entire program. The extra function wouldn't hurt much but because it's only used once and because it is essentially a one-liner I think it's cleaner to just move that one line to sqlitedb.cpp. This way we can also avoid the extra include in Settings.h.